cypress/integration/features.spec.js

```diff
@@ -0,0 +1,122 @@
+  it("Check if login features Harvard key option", () => {
+    cy.visit("/");
+    cy.get("a").contains("Sign in").click();
+    cy.get("a").contains("Continue with Harvard Key System");
+  });
+
+  // https://github.com/Harvard-ATG/scalar-plugin-harvardsettings
+  it('Checks if "Harvard Settings" is present in menu tabs', () => {
+    cy.loginScalar();
+    cy.get("a").contains("Dashboard").click();
+    cy.get("a").contains("Harvard Settings");
+    cy.get("a").contains("My account").click();
+    cy.get('input[value="New book title"]').type("Features Test Book");
+    cy.get('input[value="Create"]').click();
+  });
+  //   https://github.com/Harvard-ATG/scalar/pull/3
+  it("Allow book owner to search for users by name or email.", () => {
+    cy.loginScalar();
+    cy.get("a").contains("Dashboard").click();
+    cy.get('select[name="book_id"]').select("Features Test Book").wait(3000);
+    cy.get('a[href="#tabs-users"]').contains("Book users").click();
+    cy.get("#connect_book_user_link").contains("Add a user (admin)").click().wait(3000);
+    // lookup by email
+    cy.get('input[name="searchValue"]').type(Cypress.env('testUser')).wait(3000);
+    cy.get('div[class="results"]').contains(Cypress.env('testUser'));
+    //  lookup by user full name
+    cy.get('input[name="searchValue"]').clear()
+    cy.get('input[name="searchValue"]').type(Cypress.env('testUserFullName')).wait(3000);
+    cy.get('div[class="results"]').contains(Cypress.env('testUserFullName'));
+  });
+
+  // https://github.com/Harvard-ATG/scalar/pull/18
+  it('Checks if "My Pages" and "My Media" is present in "Categories" "Media" tab respectively', () => {
+    cy.loginScalar();
+    cy.get("a").contains("Dashboard").click();
+    cy.get("a").contains("My account").click();
+    cy.get('select[name="book_id"]').select("Features Test Book");
+    cy.wait(3000);
+    cy.get('a[href="#tabs-categories"]')
+      .contains("Categories")
+      .click({ force: true });
+    cy.get('input[id="my_pages"]').click().wait(3000)
+    cy.get('a[href="#tabs-media"]')
+      .contains("Media")
+      .click({ force: true });
+    cy.get('input[id="my_media"]').click();
+  });
+
+  //   https://github.com/Harvard-ATG/scalar/pull/16
+  it("Test Mirador fix - multiple mirador instances on the page", () => {
+    cy.loginScalar();
+    cy.get("a").contains("Dashboard").click();
+    cy.get("a").contains("My account").click();
+    cy.get('select[name="book_id"]').select("Features Test Book");
+    cy.wait(3000);
+    cy.get("a").contains("Back to book").click();
+    cy.wait(3000);
+
+    // Add example art museum manifest
+    cy.addManifest(
+      "168614",
+      "https://iiif.harvardartmuseums.org/manifests/object/168614"
+    ).wait(3000);
+    cy.addManifest(
+      "342292",
+      "https://iiif.harvardartmuseums.org/manifests/object/342292"
+    ).wait(3000);
+    // add to page
+    cy.get(
+      'a[href="https://scalar-dev.fas.harvard.edu/features-test-book/new.edit"]'
```

Comment:
Is it possible to make this independent of `scalar-dev.fas.harvard.edu` (e.g. so it works on a local installation)? For example, could you change the [attribute selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors) to something like:

```
a[href$="/features-test-book/new.edit"]
```